Update `README`

francis-zhao opened the pull request:
- Improved README doc
- Localized README doc in Mandarin Chinese (CN & TW)

paulmillr commented:
Please revert the change of table in README. It's hard to re-verify.

Otherwise, looks good for merge.

francis-zhao commented:
Sorry for the inconvenience. It should be that I used _Prettier_ as the default formatting tool for markdown, which caused the table padding spaces to be inconsistent with the original document, and makes it difficult to review changes in GitHub.

Is it convenient for you to preview changes by downloading both the original and the modified README file to the local?

I updated DNS resolvers' names referring to the corresponding links, and moved all links on `Operated`/`Independent` to the names, since the links are homepages/documents of these resolver services.

Or would it be clearer to use reference link syntax of Markdown for the table by moving all links to the bottom of the README?
```markdown
<!-- Current link syntax -->
- [Cloudflare 1.1.1.1](https://developers.cloudflare.com/1.1.1.1/encryption/)
- [Google Public DNS](https://developers.google.com/speed/public-dns/docs/secure-transports)

<!-- Reference link syntax -->
- [Cloudflare 1.1.1.1][cloudflare-dns]
- [Google Public DNS][google-dns]

[cloudflare-dns]: https://developers.cloudflare.com/1.1.1.1/encryption/
[google-dns]: https://developers.google.com/speed/public-dns/docs/secure-transports
```


![image](https://user-images.githubusercontent.com/37755859/235030780-4e5619bf-7ea0-4ae4-ad42-04303ade22a9.png)
original

![image](https://user-images.githubusercontent.com/37755859/235030982-da36fff3-2c09-4bd9-ab24-51a7b67b90d3.png)
original preview

![image](https://user-images.githubusercontent.com/37755859/235030819-3e0f52de-0450-4385-995d-ec99f54c2cfa.png)
![image](https://user-images.githubusercontent.com/37755859/235030882-eec996d9-1251-493b-b72c-fbbde7138a04.png)
modified

![image](https://user-images.githubusercontent.com/37755859/235031031-e73f4b63-00f5-473a-a314-1070c2ad043b.png)
modified preview

paulmillr commented:
On a second thought, your change looks good! Will review thoroughly.

paulmillr commented:
Do you think using <!-- Reference link syntax --> will make raw markdown better-looking?

francis-zhao commented:
As far as I am concerned, switching to reference link syntax makes it easier to maintain the README file.

![image](https://user-images.githubusercontent.com/37755859/235035746-7007de76-22b3-401c-92ad-cf2e6714d12a.png)
![image](https://user-images.githubusercontent.com/37755859/235035836-537085f6-6ea2-4e5f-917c-a7b7a029e069.png)
just like this

paulmillr commented:
abbreviation sounds good. index number meh.

francis-zhao commented:
So have you decided whether to change links of the table to this syntax, or keep it as it is?

paulmillr commented:
Your new table looks good. It will be even better with abbreviations such as on your last screenshots.

francis-zhao commented:
It's done, please check it out when you're available.

paulmillr commented:
thanks
